docker-compose.yml

```diff
@@ -10,20 +10,26 @@ services:
     ports:
       - "7896:7896"
     environment:
+#      - frontend.main-page.url=http://localhost:3000
+#      - cors.allowed-origin=http://localhost:3000
```

Comment:
Please double check, maybe this options can be uncommented as default fonfiguration?

Reply:
I left them for reference. They are not necessary if lpvs backend and frontend run in a single container.